config/application.rb

```diff
@@ -25,5 +25,9 @@ module Octobox
   class Application < Rails::Application
     config.eager_load_paths << Rails.root.join("lib")
     config.exceptions_app = routes
+
+    unless File.basename($0) == 'rake' or File.basename($0) == 'rails' or secrets.secret_key_base.length >= 32 then
```

Comment:
Can you change the `or` for `||`?

Also I don't think the `then` is needed on the end.